packages/actions-global/src/actions/show-notification.ts

```diff
@@ -19,5 +25,8 @@ export default function showNotification(notification, meta = {}) {
   if (!isNumber(dismissAfter))
     dismissAfter = notification.kind === 'success' ? 5000 : 0;
 
-  return addNotification(notification, { ...meta, dismissAfter });
+  return addNotification<Pick<Notification, 'id'>>(notification, {
```

Comment:
We only pick the `id` field (as a type) because that's the only thing that the `notifications` package is concerned with.